utils/utils.go

```diff
@@ -97,3 +93,16 @@ func SanitizeHexString(hex string) string {
 	}
 	return hex
 }
+
+func StripFromString(input, substring string) string {
```

Comment:
Leave a comment here that it removes the first occurance of the substring AND anything after it.